patterns/atoms/media/video/video.twig

```diff
@@ -0,0 +1,4 @@
+<video width="{{video.width}}" height="{{video.height}}" controls="{{video.controls}}" class="{{video.class}}">
+	<source src="{{video.src}}" type="video/ogg">
```

Comment:
I don't have time to work on specifics like this right now as my focus is on the import/export of the whole library. Would you have time to tackle this pattern and get it correct @sergesemashko?
Perhaps this version of the video pattern could be added when this PR is complete and then you could fork from that?